dummy-api/profile/overview.ts

```diff
@@ -0,0 +1,454 @@
+import PlayerTshirtIcon from '@/icons/PlayerTShirtPngIcon.png'
+import PlayerUpIcon from '@/icons/PlayerUpPngIcon.png'
+import SecondLeagueIcon from '@/icons/SecondLeague.png'
+import ThirdLeagueIcon from '@/icons/ThirdLeague.png'
+
+import { IProfileCareerData } from './career'
+import { IProfileMatchesSeason } from './matches'
+
+export interface IProfileOverviewData {
+  inShort: {
+    title: string
+    icon: string
+  }[]
+  transfers: {
+    id: number
+    from: {
+      logo: string
+      name: string
+    }
+    to: {
+      logo: string
+      name: string
+    }
+    type: string
+    date: string
+  }[]
+  similarPlayers: {
+    name: string
+    image: string
+    score: number
+    premium: boolean
+    age: number
+    scoreTrend: 'up' | 'down'
+    position: string
+    team: string
+    competition: string
+  }[]
```

Comment:
oh I'm sorry I missed this XD we have score in this format in other endpoints, let's be consistent
```suggestion
  similarPlayers: {
    name: string
    image: string
    premium: boolean
    age: number
    position: string
    team: string
    competition: string
    score: {
      value: number
      trend: 'up' | 'down'
    }
  }[]
```